utils/context.ts

```diff
@@ -1,6 +1,7 @@
-import { createContext } from 'react';
+import { createContext, useContext } from 'react';
 import { SpotlightLocation } from '.';
 import { CountryInfo } from './data';
 
 export const LocationContext = createContext<SpotlightLocation | undefined>(undefined);
 export const CountryContext = createContext<CountryInfo>({ countryCode: '', countryName: '', currencyCode: '' });
+export const useCountryContext = (): CountryInfo => useContext(CountryContext);
```

Comment:
Nope ... not a very useful change ... wherever u want to use the CountryContext, write this line instead ...

Reply:
Or maybe not ... hmm... oh well, just leave it ... but try to check if there's already an existing usage for CountryContext. Don't mind the other comments about it besides moving them.